kafkabp/config.go

```diff
@@ -138,6 +145,14 @@ func (cfg *ConsumerConfig) NewSaramaConfig() (*sarama.Config, error) {
 	// Return any errors that occurred to the Errors channel.
 	c.Consumer.Return.Errors = true
 
+	// Set the commit frequency to 5 seconds or other config value if provided
+	c.Consumer.Offsets.AutoCommit.Enable = true
+	if cfg.AutoCommitInterval != 0 {
+		c.Consumer.Offsets.AutoCommit.Interval = cfg.AutoCommitInterval * time.Second
+	} else {
+		c.Consumer.Offsets.AutoCommit.Interval = 5 * time.Second
+	}
```

Comment:
This does some subtly funky stuff as "bare" `time.Duration` values are in nanoseconds.

We want to use the value the user provides to us directly, without multiplying by `time.Second` (which [is actually the integer 1000000000](https://go.dev/play/p/HJ2PVk9zhnY)).

We also probably want to set a lower bound of 1 second so that users don't inadvertently set their commit value in nanoseconds 😅 

And as I'm writing this out I'm convincing myself we should just go back to the way that you had it originally where `AutoCommitInterval` is just a regular old `int`... (and maybe we call it `AutoCommitIntervalSeconds` for clarity)

🤷‍♂️

Reply:
Providing the value in nanoseconds seems potentially confusing to me, especially because the way you'd set this in kafka would be in milliseconds so folks familiar with that might mix them up. Adding a lower bound would help, but I think going back to an int makes the most sense and is the least error prone. The renaming you suggested sounds good, I'll do that

Reply:
fwiw you don't _have_ to

Reply:
fwiw you don't _have_ to provide the value in nanoseconds (see: https://pkg.go.dev/time#ParseDuration), that's just the default